src/utils/wallets/getMnemonicAddressName.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import { t } from 'ttag'
+
+import { checkMnemonicType } from '.'
+
+export function getMnemonicAddressName(
+  wallet: Wallet,
+  addressName: ?string,
+): string {
+  const {
+    type,
+    addressIndex,
+  }: Wallet = wallet
+
+  const isMnemonic: boolean = checkMnemonicType(type)
+  const index: number = addressIndex + 1
+
+  if (!isMnemonic) {
+    throw new Error(t`WalletInconsistentDataErrorData`)
```

Comment:
Now, you can change this into `WalletInconsistentDataError`

`JWWEB-621: Selectors, that throw`, merged :)